src/Shared/Recruit.Vacancies.Client/Infrastructure/Services/TrainingProvider/TrainingProviderService.cs

```diff
@@ -20,6 +21,9 @@ public TrainingProviderService(ILogger<TrainingProviderService> logger, IProvide
 
         public async Task<Domain.Entities.TrainingProvider> GetProviderAsync(long ukprn)
         {
+            if (ukprn == Constants.EsfaTestTrainingProviderUkprn)
```

Comment:
Can you use EsfaTestTrainingProvider class

Reply:
👍